cmake/wasi-sdk-sysroot.cmake

```diff
@@ -164,6 +164,8 @@ function(define_wasi_libc_sub target target_suffix lto)
     "${CMAKE_C_FLAGS} ${directory_cflags} ${CMAKE_C_FLAGS_${CMAKE_BUILD_TYPE_UPPER}}")
   list(JOIN extra_cflags_list " " extra_cflags)
 
+  set(libcompiler_rt_a ${wasi_resource_dir}/lib/wasm32-unknown-wasi/libclang_rt.builtins.a)
```

Comment:
i guess you need to use a correct version for the target after https://github.com/WebAssembly/wasi-sdk/pull/533